app/controllers/admin/brands_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Admin::BrandsController < Admin::BaseController
+  def new
+    @brand = Brand.new
+  end
+
+  def create
+    @brand = Brand.new(brand_params)
+    if @brand.save
+      flash[:notice] = "New Brand Created!"
+      redirect_to admin_dashboard_path
+    else
+      flash.now[:error] = @brand.errors.full_messages.join(', ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.